conda_forge_webservices/update_teams.py

```
@@ -0,0 +1,44 @@
+from git import GitCommandError, Repo
+from conda_build.metadata import MetaData
+import github
+import os
+
+from .utils import tmp_directory
+
+def update_team(org_name, repo_name):
+    if not repo_name.endswith("-feedstock"):
+        return
```

We should print some sort of message here. Maybe raise an error?

---

Though we can accomplish that by simply not adding a webhook to staged-recipes, right?

---

I originally thought of adding an organization hook, but we can do individual hooks now with conda-smithy